frontends/verilog/const2ast.cc

```diff
@@ -139,6 +139,10 @@ static void my_strtobin(std::vector<RTLIL::State> &data, const char *str, int le
 		data.resize(len_in_bits, msb);
 	}
 
+	if (len_in_bits == 0)
+		log_error("Illegal integer constant size of zero in %s:%d (IEEE 1800-2012, 5.7).\n",
```

Comment:
Shouldn't this be using log_file_error for consistency?

Reply:
Good catch, thanks!